javaparser-core/src/main/java/com/github/javaparser/ast/NodeList.java

```diff
@@ -165,6 +165,51 @@ public void add(int index, N node) {
         innerList.add(index, node);
     }
 
+    /**
+     * Inserts the node before all other nodes.
+     */
+    public NodeList<N> addToTop(N node) {
```

Comment:
What about `addAsFirst`? Or `addAtTheBeginning`?

Reply:
I took the naming of the related issue. I'm not sure what to take as a good source for naming these - what do list implementation normally use? And other node list implementations? What do parser people say?